fetch-ledger-develop-image/Dockerfile

```diff
@@ -58,10 +58,11 @@ RUN OPENSSL_SRC_DIR="/home/default/openssl" && \
     rm -rf "${OPENSSL_SRC_DIR}"
 
 # Required for integration tests
-RUN apt-get install -y python3-pip && \
-         pip3 install PyYAML && \
-         pip3 install -U fetchai-ledger-api && \
-         pip3 install -i https://test.pypi.org/simple/ fetchai-netutils==0.0.1
+RUN pip3 install PyYAML && \
+    pip3 install -U fetchai-ledger-api && \
+    pip3 install -i https://test.pypi.org/simple/ fetchai-netutils==0.0.1
+
+RUN pip3 install autopep8 cmake_format
```

Comment:
I would simply include these in the line above. This is technically not necessary but just keeps the Dockerfile more concise

Reply:
Done. I initially split them for readability.